src/ui/window/BrowserWindow.qml

```diff
@@ -262,10 +226,9 @@ FluidWindow {
         }
 
         MenuItem {
-            text: "Search in page"
+            text: "Find in page"
```

Comment:
use qsTr("Find in page") so in the future the string will be marked for translation